components/Chat/ChatPlaceholder/GroupChatPlaceholder.tsx

```diff
@@ -74,9 +76,13 @@ export function GroupChatPlaceholder({ messagesCount }: Props) {
       <View onLayout={onLayout} style={styles.chatPlaceholder}>
         {!conversation && (
           <View>
-            <ActivityIndicator style={styles.activitySpinner} />
+            {!topic && <ActivityIndicator style={{ marginBottom: 20 }} />}
             <Text style={styles.chatPlaceholderText}>
-              {translate("opening_conversation")}
+              {topic
+                ? isGroupTopic(topic)
```

Comment:
This should always be a group topic as it's in the GroupChatPlaceholder

Reply:
The check to display ChatPlaceholder or GroupChatPlaceholder is currently on `conversation?.isGroup` so it only works if the conversation was found. Do you wanna change it to `isGroupTopic(conversationTopic)` ?